database/seeds/MinecraftServiceTableSeeder.php

```diff
@@ -243,6 +243,41 @@ private function addCoreOptions()
             'startup' => null,
             'script_install' => $script,
         ]);
+
+        $script = <<<'EOF'
+#!/bin/ash
+# Forge Installation Script
+#
+# Server Files: /mnt/server
+apk update
+apk add curl openjdk8
```

Comment:
Alright, I have changed it so the installation container is set to an alpine openjdk8 container.